### ISSUE TITLE
Provide a multi-arch image

### ISSUE THREAD

**Issue by ciaranmcnulty** [cucumber/build#11]
Currently the image is x86-only, so running it on an ARM-based Mac causes it to be executed under QEMU

When the build gets to the /messages/dotnet, the emulator crashes:

```
/usr/share/dotnet/sdk/2.2.207/MSBuild.dll -bl -consoleloggerparameters:Summary -maxcpucount -property:Configuration=Release -p:SnapshotSuffix= -restore -target:Build -verbosity:m ./Cucumber.Messages.sln
qemu: uncaught target signal 6 (Aborted) - core dumped
qemu: uncaught target signal 6 (Aborted) - core dumped
make[2]: *** [default.mk:29: .built] Aborted
make[2]: Leaving directory '/app/messages/dotnet'
make[1]: *** [default.mk:17: default-dotnet/] Error 2
make[1]: Leaving directory '/app/messages'
make: *** [Makefile:24: default-messages] Error 2
```

Rather than trying to fix QEMU it would be good to provide a multi-arch image so ARM builds are possible

**Comment by mattwynne** [comment]
#12 is merged, we just need to release 0.4.0 which I hope to do via a GitHub Actions workflow when we merge #21

**Comment by mattwynne** [comment]
Released in 0.4.4
